editor/scene_tree_dock.cpp

```diff
@@ -1898,8 +1898,9 @@ void SceneTreeDock::_do_reparent(Node *p_new_parent, int p_position_in_parent, V
 		if (p_nodes[ni] == p_new_parent) {
 			return; // Attempt to reparent to itself.
 		}
+		ERR_FAIL_COND_MSG(p_nodes[ni]->get_internal_mode() != Node::INTERNAL_MODE_DISABLED, "Trying to reparent internal node, this is a bug.");
```

Comment:
*Technically* a user may add an internal node and set its owner, so it's both internal and appears in the scene tree.
I think blocking moving of internal nodes is fine, but attempting to move them is not strictly a bug.

Reply:
Also this is not needed:
```suggestion
		ERR_FAIL_COND_MSG(p_nodes[ni]->get_internal_mode() != INTERNAL_MODE_DISABLED, "Trying to reparent internal node, this is a bug.");
```
(not sure if we have any guideline for inherited enums)

Reply:
Also changed `"Trying to reparent internal node` to `"Trying to move internal node` as `_do_reparent` can act within the same parent. :upside_down_face: